core/templates/pages/story-editor-page/editor-tab/story-editor.directive.spec.ts

```diff
@@ -279,6 +279,10 @@ describe('Story editor Directive having two story nodes', function() {
     expect(storyUpdateSpy).toHaveBeenCalled();
   });
 
+  it('should check if story notes length is exceeded', function() {
+    expect($scope.isNotesLengthExceeded('story notes')).toEqual('false');
```

Comment:
Hm, should we change the length and then expect that the function returns true when the length has exceeded? That way, if there is a regression in the function, our test will catch it.